packages/keyboard-shortcuts/src/store/actions.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Returns an action object used to register a new keyboard shortcut.
+ *
+ * @param {string} name        Shortcut name.
+ * @param {string} category    Shortcut category.
+ * @param {string} description Shortcut description.
+ * @param {Object} combination Shortcut keys combination.
```

Comment:
Are we optimizing that a registered key would always be some sort of combination, and not just a single keystroke?

Reply:
I also wonder if `aliases` could be collapsed into this, where a "combination" could be an array of combinations.

Reply:
> 1 

Combination supports just a character as well if you omit the "modifier"

> 2

I had the same doubt here, We do show one combination only in the modal, so I was wondering if we should just take the first one or be more explicit.

Reply:
>I had the same doubt here, We do show one combination only in the modal, so I was wondering if we should just take the first one or be more explicit.

That we have some UI around a "canonical" form of the combination seems like a good argument to have a separate, explicit concept of aliases. 👍